k256/src/schnorr/signing.rs

```diff
@@ -78,7 +78,7 @@ impl SigningKey {
     /// The preferred interfaces are the [`Signer`] or [`RandomizedSigner`] traits.
     pub fn sign_prehash_with_aux_rand(
```

Comment:
Perhaps this method should be renamed to reflect the input is no longer necessarily a prehash?

Something like `sign_raw` perhaps?

Reply:
Yeah makes sense, I renamed to `sign_raw`. I didn't know if you wanted to keep any compatability forwarding (eg a `#[deprecated]` `sign_prehash_with_aux_rand` that forwards to `sign_raw`), so I left it out for now.

Reply:
That's fine. We can make a note in the changelog.